olmo/checkpoint.py

```diff
@@ -1376,12 +1374,12 @@ def save_checkpoint(
                 save_overwrite=self.cfg.save_overwrite,
             )
 
-            # Save config.
-            self._save_config(checkpoint_dir, upload_to=upload_to)
-
             # Save metadata.
             self._save_metadata(checkpoint_dir, upload_to=upload_to)
 
+            # Save config.
+            self._save_config(checkpoint_dir, upload_to=upload_to)
```

Comment:
Maybe add a comment explaining why this is now last?

Reply:
c6c114d7